code-challenges.test.js

```diff
@@ -12,20 +12,67 @@
 
 // --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
 
-// a) Create a test with expect statements for each of the variables provided.
+//Psuedo Code: Create a function called lengthOf that takes in a number greater than 2 and the Fibonaacci sequence numbers.
+// create a function which outputs an array that is the fib sequence to the length given
+// Input: an array of numbers.
+//Output: Each number of the array added to the next following the Fibonaacci sequence.
+// Create a variable called arrayLength that will be used to set the starting point of the addition. (question? If  I was to use  let numlength = [0, 0] would that cause the code to add the number at the 0 index twice?)
+// create a for loop that will cause the function to continue counting through each array until it reaches a stopping point.
+// push the sum of the numbers to the next iteration using .push
+// return the new array
 
+// a) Create a test with expect statements for each of the variables provided.
+describe("arrayLength", () => {
+  it("returns an array with the numbers greater than 2", () => {
+    expect(arrayLength + (fibLength1)).toEqual[1, 1, 2, 3, 4, 8]
+    expect(arrayLength + (fibLength2)).toEqual[1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
```

Comment:
Almost got it! You don't need the + in between the function and argument. And after toEqual you will have () with the array inside.

Reply:
Thank you for that feedback, I went over this with Chris Lee the tutor Elyse set me up with and he helped me understand how the syntax for this question goes.